python/tests/test_xarray_backend.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+xr = pytest.importorskip("xarray")
+
+
+def test_xarray_backend_gefs_ensemble():
+    """Test reading GEFS ensemble average file using xarray backend"""
+    # Open the GEFS ensemble average file using the gribberish backend
+    ds = xr.open_dataset(
+        "./../gribberish/tests/data/geavg.t12z.pgrb2a.0p50.f000",
+        engine="gribberish"
+    )
+
+    # Verify dataset was loaded
+    assert ds is not None
+    assert len(ds.data_vars) > 0
+
+    # Check that we can access data variables
+    for var_name in ds.data_vars:
+        var = ds[var_name]
```

Comment:
Can we change this test to check for exact variables that are expected?or dimension sizes?